tests/wpt/mozilla/tests/css/get-computed-style-for-url.html

```diff
@@ -40,6 +40,29 @@
     }, `getComputedStyle(elem) for url() ${property} uses the resolved URL and elem.style uses the original URL`);
   }
 
+  function testNoneForProperty(property) {
+    test(function() {
+      var elem = document.createElement("div");
+      elem.style[property] = "none";
+      container.appendChild(elem);
+
```

Comment:
I finally managed to write a tests that failed with my previous implementation. In this case, `getComputedStyle` would return `""` and `elem.style` would return `"none"` without fixing parsing for `ListStyleImage`.

Now, all the new tests pass.

Reply:
Nice!